pkg/loop/process_test.go

```diff
@@ -0,0 +1,25 @@
+package loop_test
+
+import (
+	"os/exec"
+
+	"github.com/smartcontractkit/chainlink-relay/pkg/loop/internal/test"
+)
+
+type HelperProcessCommand struct {
+	test.HelperProcessCommand
+}
+
+func (h HelperProcessCommand) New() *exec.Cmd {
+	h.CommandLocation = "./internal/test/cmd/main.go"
+	return h.HelperProcessCommand.New()
+}
+
+func NewHelperProcessCommand(command string) *exec.Cmd {
+	h := HelperProcessCommand{
+		HelperProcessCommand: test.HelperProcessCommand{
+			Command: command,
+		},
+	}
+	return h.New()
+}
```

Comment:
Oh This can be simplified a little further to drop the nested type:
```suggestion
type HelperProcessCommand test.HelperProcessCommand

func (h HelperProcessCommand) New() *exec.Cmd {
	h.CommandLocation = "./internal/test/cmd/main.go"
	return (test.HelperProcessCommand)(h).New()
}

func NewHelperProcessCommand(command string) *exec.Cmd {
	h := HelperProcessCommand{
		Command: command,
	}
	return h.New()
}
```

Reply:
😥 I've refactored again, but want to point out gently that the first version (to use embedding) was your suggestion pretty much verbatim.

Reply:
What do you mean? Isn't this strictly an improvement on that version?